sklearn/feature_selection/from_model.py

```diff
@@ -173,6 +174,10 @@ class SelectFromModel(BaseEstimator, SelectorMixin):
         Otherwise train the model using ``fit`` and then ``transform`` to do
         feature selection.
 
+    norm_order : non-zero int, inf, -inf, default 1
+        Order of the norm used to compare the vectors of coefficients in the
+        case where the coeff_ attribute of the estimator is of dimension 2.
```

Comment:
I think we can just use the word features here since there can also be `feature_importances_`